components/server/src/analytics.ts

```diff
@@ -34,6 +34,7 @@ export async function trackSignup(user: User, request: Request, analytics: IAnal
         properties: {
             auth_provider: user.identities[0].authProviderId,
             qualified: !!request.cookies["gitpod-marketing-website-visited"],
+            blocked: user.blocked,
```

Comment:
@jakobhero I added this property. Does that require any work from your side?

Reply:
hi @geropl, thanks for looping me in! this does not cause any issues downstream, i will update the tracking plan with the additional property.